saleor/static/dashboard-next/components/AppLayout/AppLayout.tsx

```diff
@@ -91,16 +91,29 @@ const styles = (theme: Theme) =>
         marginTop: 12,
         transform: "rotate(180deg)"
       },
+      "&:hover": {
+        background: "#E6F3F3"
+      },
       background: theme.palette.background.paper,
+      border: `solid 1px #EAEAEA`,
       borderRadius: "50%",
       cursor: "pointer",
       height: 32,
       position: "absolute",
       right: -16,
       top: 65,
+      transition: "background 0.2s",
```

Comment:
Shouldn't we use a variable for the transition time value? It's used in a few places and I guess we always want to use the same value for all of them?